client/src/pages/Background.js

```diff
@@ -0,0 +1,306 @@
+import React, { Component } from "react";
+
+import {
+  Typography,
+  Grid,
+  Paper,
+  Button,
+  Box,
+  TextField,
+  Snackbar,
+  FormControl,
+  Select,
+  InputLabel,
+  MenuItem,
+} from "@material-ui/core";
+import MuiAlert from "@material-ui/lab/Alert";
+import Rating from '@material-ui/lab/Rating';
+import { withStyles } from "@material-ui/core/styles";
+import { Route, Link } from "react-router-dom";
+import pic1 from "../assets/pic1.png";
+
+function Alert(props) {
+  return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+
+const loginStyle = (theme) => ({
+  root: {
+    flexGrow: 1,
+    height: "100vh",
+  },
+  paper: {
+    padding: theme.spacing(0),
+    margin: "auto",
+    width: "Full",
+    height: "100%",
+    border: 0,
+  },
+  image: {
+    //    width: '90vh',
+    height: "100vh",
+  },
+  img: {
+    margin: "auto",
+    display: "block",
+    maxWidth: "100%",
+    maxHeight: "100%",
+  },
+  text: {
+    paddingTop: "0.5rem",
+    paddingLeft: "20%",
+    paddingRight: "20%",
+  },
+  account: {
+    paddingTop: "0.5rem",
+  },
+  button: {
+    borderRadius: 35,
+    marginLeft: "1rem",
+    marginRight: "1.5rem",
+    marginBottom: "2rem",
+    paddingLeft: "1.5rem",
+    paddingRight: "1.5rem",
+  },
+  title: {
+    color: "#0000ff",
+    fontFamily: "Arial",
+    marginTop: "3rem",
+    paddingBottom: "1rem",
+  },
+  input: {
+    marginTop: "0.5rem",
+    marginBottom: "1rem",
+  },
+  nextStep: {
+    backgroundColor: "#0000ff",
+    color: "#ffffff",
+    borderRadius: 35,
+    marginTop: "3rem",
+    paddingLeft: "1.5rem",
+    paddingRight: "1.5rem",
+  },
+  question: {
+    marginTop: "2rem",
+    marginBottom: "0.5rem",
+  }
+});
+const labels = {
+  1: "JUNIOR",
+  2: "ADVANCED JUNIOR",
+  3: "INTERMEDIATE",
+  4: "SENIOR",
+  5: "ADVANCED SENIOR",
+}
+const details = {
+  1: "Do not have any experience on interviews",
+  2: "Had one or two interviews before",
+  3: "Had a few job interviews, but need more practice",
+  4: "Had seven to twelve interviews before, and want to know more about interview skills",
+  5: "Had more than twelve interviews before, and know much about interview skills",
+}
+
+class Login extends Component {
```

Comment:
i notice a use of class based components - may want to check in with team lead for this vs using functional and hooks

Reply:
also, is this component meant to be called "Background" what's the difference between this and Login?

Reply:
Yes we should be using functional components - and this should be called Background 👍

Reply:
Transform to functional components.